### DISCUSSION TITLE
Any way to reduce the size ?

### DISCUSSION THREAD

**Discussion by mderazon** [markedjs/marked#3308] [Q&A]
If I want to use a small subset of markdown, can I reduce the size of the library in some way ?

**Comment by UziTech** [comment]
This sounds like a better question for stack overflow. Other than manually removing the parts that you don't need I don't know of any software that will automatically remove the parts you don't use.

**Reply by mderazon** [reply]
Yeah I could fork it and remove the code I don't use but I was asking if there's a built in way the library like tree shaking for the parts that you don't use.

I take it as a no then
Thanks

**Reply by UziTech** [reply]
Marked converts markdown to html that doesn't have anything to do with tree shaking. Maybe using webpack or another bundler can do some tree shaking.

It is unlikely to help though since marked only does one thing it is not like other utility libraries that has a bunch of code that isn't used if you only use part of it.
